test/verify/check-storage-mdraid

```diff
@@ -172,12 +182,12 @@ class TestStorageMdRaid(StorageCase):
         # Stopping and starting the array will create the expected
         # symlink in /dev/md/.
         #
-        b.wait_text(info_field("Device", m.image), "/dev/md/ARR")
+        b.wait_text(info_field("Device", m.image), "/dev/md/raid0")
 
         # Partitions also get symlinks in /dev/md/, so they should
-        # have names like "/dev/md/ARR1".
+        # have names like "/dev/md/raid01".
```

Comment:
That would be `/dev/md/raid0p1`, I believe.

Reply:
It indeed is!
but why it was `ARR` before and now it has that one `p`?

Reply:
Because `/dev/md/raid0` ends in a digit.